mw-webapp/src/logic/landingPages/mentorsLandingPage/MentorsLandingPage.module.scss

```diff
@@ -0,0 +1,98 @@
+@import "src/styles/variables";
+
+$accordionMaxWidth: 1032px;
```

Comment:
let's play with this var. Is it okey if we will make accordion width - 100%? Or it's too much?

Reply:
100% looks fine)